shared/_static/home.css

```diff
@@ -0,0 +1,8 @@
+/*
+  For use in /core only (see /core/conf.py)
+  Hides sphinx generated homepage header (https://docs.tenstorrent.com/)
+*/
+
+#tenstorrent > h1 {
```

Comment:
Can I not remove it via deleting the first two lines of `index.rst` 
```
Tenstorrent
==============================
```

Reply:
> ```
> Tenstorrent
> ==============================
> ```

Removing this seems to make the page title default to the closest header:

![Screenshot 2024-11-25 at 10 56 50 AM](https://github.com/user-attachments/assets/cf0c92ef-9d7b-4e5d-adb7-288c22b1a816)